### PR TITLE
fixes https://github.com/wso2/docs-apim/issues/4269

### PR DESCRIPTION
**Pull request by Mariangela**
## Purpose
- fixes https://github.com/wso2/docs-apim/issues/4269
- updating observe related overviews

### CODE REVIEW

File: en/docs/install-and-setup/setup/mi-setup/observability/observability-deployment-strategy.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ The cloud-native solution is more suitable in the following scenarios:
 
 ### Technologies
 
-The cloud-native observability solution is based on proven projects from the **Cloud Native Computing Foundation**, which makes the solution cloud native and future proof. Following are the technologies used in the current solution:
+The cloud-native observability solution is based on proven projects from the **Cloud Native Computing Foundation**, which makes the solution cloud native and future-roof. Following are the technologies used in the current solution:
```
Comment (Mariangela) [inline en/docs/install-and-setup/setup/mi-setup/observability/observability-deployment-strategy.md:20]:
```suggestion
The cloud-native observability solution is based on proven projects from the **Cloud Native Computing Foundation**, which makes the solution cloud native and future-proof. Following are the technologies used in the current solution:
```

File: en/docs/observe/micro-integrator/cloud-native-observability-overview.md [hunk 2]
```diff
@@ -4,11 +4,35 @@ title: Micro Integrator Observability Overview - WSO2 API Manager 4.0.0
 
 # Micro Integrator Observability Overview
 
-This section explains how to set up the observability solutions and perform management tasks for the Micro Integrator.
+The following diagram depicts the complete **cloud native** observability solution for your Micro Integrator deployment, which includes **metrics monitoring**, **log monitoring**, and **message tracing** capabilities.
 
-### Observability solutions
+![Cloud Native Deployment Architecture]({{base_path}}/assets/img/integrate/monitoring-dashboard/cloud-native-deployment-architecture.png)
```
Comment (Mariangela) [inline en/docs/observe/micro-integrator/cloud-native-observability-overview.md:9]:
```suggestion
[![Cloud Native Deployment Architecture]({{base_path}}/assets/img/integrate/monitoring-dashboard/cloud-native-deployment-architecture.png)]({{base_path}}/assets/img/integrate/monitoring-dashboard/cloud-native-deployment-architecture.png)
```

File: en/docs/observe/micro-integrator/cloud-native-observability-overview.md [hunk 3]
```diff
@@ -4,11 +4,35 @@ title: Micro Integrator Observability Overview - WSO2 API Manager 4.0.0
...
 
-There are two cloud-native observability solutions for the Micro Integrator; The Kubernetes based deployment and the VM based deployment.
+## Minimum cloud native observability
+
+The basic deployment offers you metrics capabilities. You can set up the basic deployment with only Prometheus and Grafana to view and explore with the available Prometheus metrics.
+
+![Cloud Native Deployment - Minimum]({{base_path}}/assets/img/integrate/monitoring-dashboard/cloud-native-observability-metrics.png)
```
Comment (Mariangela) [inline en/docs/observe/micro-integrator/cloud-native-observability-overview.md:15]:
```suggestion
[![Cloud Native Deployment - Minimum]({{base_path}}/assets/img/integrate/monitoring-dashboard/cloud-native-observability-metrics.png)]({{base_path}}/assets/img/integrate/monitoring-dashboard/cloud-native-observability-metrics.png)
```

File: en/docs/observe/micro-integrator/cloud-native-observability-overview.md [hunk 4]
```diff
@@ -4,11 +4,35 @@ title: Micro Integrator Observability Overview - WSO2 API Manager 4.0.0
...
+
+## Cloud native observability add ons
+
+You can also set up different flavours of the observability solution depending on your requirement.
+
+### Log processing add on
+ 
+Once you set up the basic deployment, you can integrate log-processing capabilities. To use this, you need to install **Fluent-Bit** as the logging agent and **Grafana Loki** as the log aggregator.
+
+![Cloud Native Deployment with Logs]({{base_path}}/assets/img/integrate/monitoring-dashboard/cloud-native-observability-logs.png)
```
Comment (Mariangela) [inline en/docs/observe/micro-integrator/cloud-native-observability-overview.md:25]:
```suggestion
[![Cloud Native Deployment with Logs]({{base_path}}/assets/img/integrate/monitoring-dashboard/cloud-native-observability-logs.png)]({{base_path}}/assets/img/integrate/monitoring-dashboard/cloud-native-observability-logs.png)
```

File: en/docs/observe/micro-integrator/cloud-native-observability-overview.md [hunk 5]
```diff
@@ -4,11 +4,35 @@ title: Micro Integrator Observability Overview - WSO2 API Manager 4.0.0
...
+
+### Message tracing add on
+
+Once you set up the basic deployment, you can integrate message tracing capabilities. To use this you need to install **Jaeger**.  
+
+![Cloud Native Deployment with Tracing]({{base_path}}/assets/img/integrate/monitoring-dashboard/cloud-native-observability-tracing.png)
```
Comment (Mariangela) [inline en/docs/observe/micro-integrator/cloud-native-observability-overview.md:31]:
```suggestion
[![Cloud Native Deployment with Tracing]({{base_path}}/assets/img/integrate/monitoring-dashboard/cloud-native-observability-tracing.png)]({{base_path}}/assets/img/integrate/monitoring-dashboard/cloud-native-observability-tracing.png)
```

File: en/docs/install-and-setup/setup/mi-setup/observability/observability-deployment-strategy.md [hunk 6]
```diff
@@ -17,7 +17,7 @@ The cloud-native solution is more suitable in the following scenarios:
 
 ### Technologies
 
-The cloud-native observability solution is based on proven projects from the **Cloud Native Computing Foundation**, which makes the solution cloud native and future proof. Following are the technologies used in the current solution:
+The cloud-native observability solution is based on proven projects from the **Cloud Native Computing Foundation**, which makes the solution cloud native and future-proof. Following are the technologies used in the current solution:
```
Comment (samgnaniah) [inline en/docs/install-and-setup/setup/mi-setup/observability/observability-deployment-strategy.md:20]: Shall we reword "future-proof" to something else? Let's use something like "sustainable" or "not susceptible to changes in future trends".

**Review by samgnaniah — APPROVED** [review]

File: en/docs/install-and-setup/setup/mi-setup/observability/observability-deployment-strategy.md (hunk 6 above)
Comment (Mariangela) [inline en/docs/install-and-setup/setup/mi-setup/observability/observability-deployment-strategy.md:20]:
```suggestion
The cloud-native observability solution is based on proven projects from the **Cloud Native Computing Foundation**, which makes the solution cloud native and not susceptible to changes in future trends. Following are the technologies used in the current solution:
```

**Pull request merged**